src/cpu/jit_uni_eltwise.cpp

```diff
@@ -608,6 +608,25 @@ void jit_uni_eltwise_injector_f32<isa>::swish_compute_vector(
     h->uni_vmulps(vmm_src, vmm_src, vmm_aux0);
 }
 
+template <cpu_isa_t isa>
+void jit_uni_eltwise_injector_f32<isa>::hswish_compute_vector(
+        const Vmm &vmm_src) {
+    // Save src data on stack for later usage
+    h->sub(h->rsp, vlen);
+    h->uni_vmovups(h->ptr[h->rsp], vmm_src);
+    // x + 3
+    h->uni_vaddps(vmm_src, vmm_src, table_val(1));
+    // relu6(x + 3)
+    h->uni_vmaxps(vmm_src, vmm_src, table_val(0));
+    h->uni_vminps(vmm_src, vmm_src, table_val(2));
+    // relu6(x + 3) / 6
+    h->uni_vdivps(vmm_src, vmm_src, table_val(2));
```

Comment:
Division is a very expensive operation. I suggest to replace it with multiplication